pom.xml

```diff
@@ -311,6 +306,12 @@
             <artifactId>spring-security-oauth2</artifactId>
             <version>${spring.security.oauth.version}</version>
          </dependency>
+
+		<dependency>
+			<groupId>com.github.zafarkhaja</groupId>
+			<artifactId>java-semver</artifactId>
+			<version>0.9.0</version>
+		</dependency>
```

Comment:
How is this being used. The library has not been developed since 2015

Reply:
This is being used for performing semver comparisons when querying the API specific versions.

The library has not been developed - I think this would be OK since the it doesn't have a *bug* on the issue list & doesn't require active development since Semver policy does not change. However, I agree that if the library uses any outdated dependencies, technology or security updates, then this would require active development